### ISSUE TITLE
[feature request] Open python file from remote server

### ISSUE THREAD

**Issue by Jakobhenningjensen** [spyder-ide/spyder#9795]
Since I can connect to an existing kernel via SSH, can you make it possible to open python files from remote servers, avoid having a local script?

**Comment by ccordoba12** [comment]
Not right now. We plan to do this in future Spyder versions.

**Comment by jsh9** [comment]
I, too, would like to see this feature implemented. FYI vscode has already implemented this feature: https://code.visualstudio.com/docs/remote/ssh

**Comment by jsh9** [comment]
Thank you in advance! @ccordoba12

**Comment by ccordoba12** [comment]
This is something we're already discussing how to implement and it'll probably land next year.

**Comment by seyeeet** [comment]
any update on this? :)

**Comment by kyle662606957** [comment]
Any update on this topic? I like very much the variable explorer in Spyder, I really hope that we can debug codes on the remote server with Spyder

**Comment by krishnaap** [comment]
@ccordoba12 Really appreciate it if that would be possible, taking a local copy of the script and the entire data every time is a bit time taking.

**Comment by ccordoba12** [comment]
We're evaluating how to improve our support for remote development. This year we asked for funding to do precisely that, but unfortunately we didn't get it. We'll see what we can do about it in 2022.

**Comment by ccordoba12** [comment]
Yes, it is. It should be available in Spyder next year (can't say exactly when).

**Comment by prishasatwani** [comment]
Hi, is there any update on when this will be available? Thanks!

**Comment by ccordoba12** [comment]
It'll be added to Spyder 6.1.0, which will be released during the second half of this year.
